BE/README.md

```diff
@@ -64,3 +64,10 @@ If you discover a security vulnerability within Laravel, please send an e-mail t
 ## License
 
 The Laravel framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
+
+# Several project settings
```

Comment:
Instructions are incomplete.
The test task specified using Docker Sail for running the project locally.

Reply:
I agree. I haven't used Sail before, and since I had tasks on my current project I didn't have time to get familiar with it properly and implement everything as needed. However, if we were to proceed working together, I would surely learn how to use Sail.